### PR TITLE
Allow setting custom notification handler

### PR DESCRIPTION
**Pull request by samiuelson**
### Description

* Allow setting custom notification handler in `Chat.Builder`
* Update Offline Support to the [last version](https://github.com/GetStream/stream-chat-android-livedata/releases/tag/0.8.6)

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog updated with client-facing changes
- [x] Reviewers added

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # To be released:
 - Fix a crash related to behaviour changes in 1.3.0-alpha08 of the AndroidX Fragment library
 - Replace Glide with Coil in AttachmentViewHolderMedia (Fix GIFs loading issues)
+- Update Offline Support to the [last version](. See changes: )https://github.com/GetStream/stream-chat-android-livedata/releases/tag/0.8.6)
```
Comment (zsmb13) [inline CHANGELOG.md:4]: The link (?) in this text is broken

File: sample/src/main/kotlin/io/getstream/chat/sample/feature/login/LoginFragment.kt [hunk 2]
```diff
@@ -28,12 +29,22 @@ class LoginFragment : Fragment(R.layout.fragment_login) {
...
                     is State.Error -> showErrorMessage(it.errorMessage)
+                    is State.RedirectToChannel -> redirectToChannel(it.cid)
                 }
             }
         )
+        activity?.intent?.apply {
+            getStringExtra(EXTRA_CHANNEL_ID)?.let {
+                viewModel.targetChannelDataReceived(it)
+            }
+        }
```
Comment (zsmb13) [inline sample/src/main/kotlin/io/getstream/chat/sample/feature/login/LoginFragment.kt:43]:
```suggestion
        activity?.intent?.getStringExtra(EXTRA_CHANNEL_ID)?.let { cid ->
            viewModel.targetChannelDataReceived(cid)
        }
```

**Review by zsmb13 — APPROVED** [review]
LGTM 👍

File: CHANGELOG.md (hunk 1 above)
Comment (samiuelson) [reply CHANGELOG.md]: Fixed 👍🏼

**Review by zsmb13 — APPROVED** [review]

**Review by JcMinarro — APPROVED** [review]

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**